scripts/tests/sanitycheck/conftest.py

```diff
@@ -33,18 +33,34 @@ def testsuite_obj(test_data, testcases_dir, tmpdir_factory):
     suite = TestSuite(board_root, testcase_root, outdir)
     return suite
 
-@pytest.fixture
-def all_testcases_dict(class_testsuite):
-    """ Pytest fixture to call add_testcase function of Testsuite class and return the testcases in kernel"""
+@pytest.fixture(name='all_testcases_dict')
+def testcases_dict(class_testsuite):
+    """ Pytest fixture to call add_testcase function of
+	Testsuite class and return the testcases in kernel"""
```

Comment:
what you mean with "testcases in kernel" ? Somehow this is filtering only tests targeting Zephyr's kernel ? That is not what I understood based on function name

Reply:
My bad. This should be reworded.  It is not related to zephyr testcases. Will correct this and thanks for noticing this.

Reply:
Addressed